templates/reconciliation.js

```diff
@@ -35,6 +35,48 @@ var vote = function vote($choice){
   nextPairing($pairing);
 }
 
+// This function looks through allVotes() to see if there are any IDs
+// matched with multiple UUIDs or (conversely) any UUIDs matched with
+// multiple IDs. If so, it returns HTML that warns about these
```

Comment:
Yeah, I noticed after making the PR that the description in the spreadsheet actually asks for much more work than the GitHub issue (i.e. it also wants an interface for unmatching or representing those matches). I can certainly split out a function to return duplicates separate from the HTML generation, but I'd be tempted to create a new issue for the more involved work of offering an interface for fixing them. (And possibly only work on that after introducing some tests for the reconciler - doing anything more complex without them would feel brittle, since it's awkward to test by hand anyway.)